data/json/items/gun/44.json

```diff
@@ -1,5 +1,34 @@
 [
   {
+    "id": "cane_rifle",
+    "type": "GUN",
+    "name": "walking cane",
+    "description": "Gun in a cane.  A custom-built .44 Magnum rifle carefully integrated into a walking cane.  It's also reinforced for hand-to-hand combat.  Never underestimate a handicapped person.",
+    "weight": 950,
+    "volume": 4,
+    "price": 400000,
+    "bashing": 15,
+    "to_hit": 2,
+    "material": [ "steel", "aluminum", "wood" ],
+    "symbol": "/",
+    "color": "dark_gray",
+    "skill": "rifle",
+    "ammo": "44",
+    "ranged_damage": 11,
```

Comment:
What is the upper limit of the Barrel Length damage bonus? The docs make it seem as if it is limitless.